db/app.js

```diff
@@ -23,6 +24,8 @@ app.get("/api/articles", getArticles);
 
 app.get("/api/articles/:article_id/comments", getCommentsByArticleId);
 
+app.post("/api/articles/:article_id/comments", addCommentByArticleId);
+
 //last end point
 app.all("*", (request, response, next) => {
```

Comment:
This error code may not be the most accurate, you'd probably want to provide a 404 Not Found message, but be cautious as this may break some of your tests